Src/Fido2.Models/AssertionOptions.cs

```diff
@@ -36,7 +36,7 @@ public class AssertionOptions : Fido2ResponseBase
     /// This OPTIONAL member contains a list of PublicKeyCredentialDescriptor objects representing public key credentials acceptable to the caller, in descending order of the caller’s preference(the first item in the list is the most preferred credential, and so on down the list)
     /// </summary>
     [JsonPropertyName("allowCredentials")]
-    public IEnumerable<PublicKeyCredentialDescriptor> AllowCredentials { get; set; }
+    public IReadOnlyList<PublicKeyCredentialDescriptor> AllowCredentials { get; set; } = new List<PublicKeyCredentialDescriptor>();
```

Comment:
@iamcarbon a.k.a Mr Zero Allocations, do we want to leave this as null? And if we do want to set a value, should it be something more like Array.Empty?

Reply:
I agree with you, this allocation is not necessary. I replaced it `Array.Empty`.